### DISCUSSION TITLE
Pada Rpubs Capstone ML, rubrik menyebut data test. Ini beneran diminta tinjauan data test atau maksudnya data validation?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#157] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/95233522/175574717-77a973c3-e99c-48a7-82e9-1fa3a66171da.JPG)
Kalau tweet yang incorectly predicted pada data test gimana? Kan student nggak punya labelnya, data test itu kan unseen data? Apakah maksudnya di validation data?

**Comment by kevbow** [comment]
Halo @Hertzspr, 

Data yang tidak berlabel ini ya nantinya yang disubmit hasil prediksinya ke leaderboard.
Nantinya hasil submition Anda akan dibandingkan dengan label yang kami miliki. Dari situlah rubrics nilai di atas didapatkan nilainya. Jangan lupa untuk melampirkan hasil screenshot-nya di report yang Anda buat.
Demikian yang dapat kami sampaikan, terima kasih.

**Comment by nisa-basalamah** [comment, marked as answer]
Halo @Hertzspr,

Terkait hal yang disebutkan pada rubrik di atas, betul, data yang digunakan adalah validation data untuk interpretasi tweets yang salah diprediksi.

Demikian yang dapat kami sampaikan. Terima kasih.

**Reply by Hertzspr** [reply]
waduh, telat nih mbak :(, udah kesubmit htmlnya jadinya cuma nebak nebak kategori (TP, TN, FP, FN) yang salah dari hasil yang merah/ijo di scoreboard.
